tests/tests.py

```diff
@@ -48,21 +48,43 @@ def test_returns_true_with_valid_date_no_level(self):
 
     def test_returns_true_with_valid_date_and_level(self):
         response = self.client.get(reverse('edtf:results_json'), {'date': '2013-12-12',
-                                                                  'level': '0'})
+                                                                  'levelFeature': '0'})
         self.assertEqual(json.loads(response.content), {'validEDTF': True})
 
     def test_returns_400_with_valid_date_and_invalid_level(self):
-        response = self.client.get(reverse('edtf:results_json'), {'date': '2013', 'level': '3'})
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013',
+                                                                  'levelFeature': '3'})
         self.assertEqual(response.status_code, 400)
 
     def test_returns_400_with_valid_date_and_empty_level(self):
-        response = self.client.get(reverse('edtf:results_json'), {'date': '2013', 'level': ''})
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013',
+                                                                  'levelFeature': ''})
         self.assertEqual(response.status_code, 400)
 
     def test_returns_false_with_invalid_date(self):
         response = self.client.get(reverse('edtf:results_json'), {'date': '2013-23-????'})
         self.assertEqual(json.loads(response.content), {'validEDTF': False})
 
+    def test_returns_true_with_valid_date_and_levelConforms_1(self):
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013/2014',
+                                                                  'levelConforms': '1'})
+        self.assertEqual(json.loads(response.content), {'validEDTF': True})
+
+    def test_returns_true_with_valid_date_and_levelConforms_2(self):
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013-12-12',
+                                                                  'levelConforms': '2'})
+        self.assertEqual(json.loads(response.content), {'validEDTF': True})
+
+    def test_returns_400_with_valid_date_and_invalid_levelConforms(self):
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013',
+                                                                  'levelConforms': '3'})
+        self.assertEqual(response.status_code, 400)
+
+    def test_returns_400_with_valid_date_and_empty_levelConforms(self):
+        response = self.client.get(reverse('edtf:results_json'), {'date': '2013',
+                                                                  'levelConforms': ''})
+        self.assertEqual(response.status_code, 400)
+
```

Comment:
I feel like we could use some additional tests. Maybe a test with a valid date against levelConforms 0? And how about some tests against levelConforms where the date is invalid?

Reply:
sure! I can add those. I think I can add some invalid dates to levelFeature url too.

Reply:
Seems like some parametrization could be worthwhile, though I suppose it could get confusing...